internal/app/machined/pkg/runtime/v1alpha1/bootloader/grub/constants.go

```diff
@@ -13,6 +13,9 @@ const (
 	// BootB is a bootloader label.
 	BootB BootLabel = "B"
 
+	// BootWipe is a bootloader label.
+	BootWipe BootLabel = "Wipe"
```

Comment:
My only concern is the name of the label, as it starts the user-visible string in GRUB menu.

What will be the final menu entry?

```
Reset Talos installation and return to maintenance mode
```

It will probably start with `Reset` ? I think we don't use the word `Wipe` often, and here we talk about a partial "wipe"